src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: Work with variables
+description: aaa
+meta_tags: deploy, orchestrator
+namespace: docs_deploy_work_with_variables
+menu_namespace: deployMenu
+permalink: /documentation/products/guides/deploy/work-with-variables/
+---
+
+During the deployment process on Edge Orchestrator, we have the possibility to work with variables. A variable can be set on an [edge service](#edge-services-variables) and on an [edge node](#edge-node).
```

Comment:
```suggestion
During the deployment process on **Edge Orchestrator**, you can work with variables. A variable can be set on an [edge service](#edge-services-variables) and an [edge node](#edge-node).
```

Reply:
A variable can be set on an [edge service](#edge-services-variables) or an [edge node](#edge-node).
OU
 A variable can be set on an [edge service](#edge-services-variables) and an [edge node](#edge-node).

Reply:
and @MarianaAguilera